File: BaekJoon_C/14681/14681/14681.c
#define _CRT_SECURE_NO_WARNINGS 
#include <stdio.h>

int main() {
	int x, y;
	int pos;

	scanf("%d", &x);
	scanf("%d", &y);

	if (x > 0 && y > 0) pos = 1;
	if (x < 0 && y > 0) pos = 2;
	if (x < 0 && y < 0) pos = 3;
	if (x > 0 && y < 0) pos = 4;

	printf("%d", pos);

	return 0;
}

File: python/BaekJoon_python/10000~19999/14000~14999/14681.py
x = int(input())
y = int(input())

if (x > 0 and y > 0) : pos = 1
elif (x < 0 and y > 0) : pos = 2
elif (x < 0 and y < 0) : pos = 3
else : pos = 4

print(pos)